psi4/src/psi4/ccresponse/polar.cc

```diff
@@ -137,11 +139,21 @@ void polar(void)
     trace[i] = tensor[i][0][0] + tensor[i][1][1] + tensor[i][2][2];
     outfile->Printf( "\n\talpha_(%5.3f) = %20.12f a.u.\n", params.omega[i], trace[i]/3.0);
 
-    // Need to generalize this for multi-wavelength calcs
-    if (params.wfn == "CC2")
-      Process::environment.globals["CC2 DIPOLE POLARIZABILITY"] = trace[i]/3.0;
-    else
-      Process::environment.globals["CCSD DIPOLE POLARIZABILITY"] = trace[i]/3.0;
+    /* make sure omega in nm is rounded */
+    long omega_nm_rd = std::lround(omega_nm);
+
+    if (params.wfn == "CC2"){
+      /* build up a string */
+      std::stringstream tag;
+      tag << "CC2 DIPOLE POLARIZABILITY @ " << omega_nm_rd << "NM";
+      Process::environment.globals[tag.str()] = trace[i]/3.0;
+    }
+    else if(params.wfn == "CCSD"){
+      /* build up a string */
+      std::stringstream tag;
+      tag << "CCSD DIPOLE POLARIZABILITY @ " << omega_nm_rd << "NM";
+      Process::environment.globals[tag.str()] = trace[i]/3.0;
+    }
   }
```

Comment:
For all of these, please make a comment section like https://github.com/psi4/psi4/blob/master/psi4/src/psi4/detci/opdm.cc#L644-L666 so the autodocs with catch them. Lowercase for variable (non-fixed) parts of the string. And note the nonstandard comment markers.

Reply:
Done, let me know if I missed anything!